### ISSUE TITLE
List with books to continue disappearing when App can reach the server

### ISSUE THREAD

**Issue by asdil12** [advplyr/audiobookshelf-app#1084]
### Expected behaviour
List with books to continue is shown when opening app.
Always at the top as it is the most important thing, as the user is very likely opening the app to continue listening to their current audiobook.

### Actual behaviour
List is only shown until the app fetched some data from the server - then it disappears completely .

When the phone is offline, the current books are shown.

### Environment data

Audiobookshelf Version:

- [x] Android App

**Comment by advplyr** [comment]
What version of the android app are you using?

**Comment by asdil12** [comment]
0.9.72-beta

**Comment by advplyr** [comment]
"Continue Books" is used in offline mode. "Continue Listening" is used when you connect to the server. At the top once you connect to the server is "Continue Listening" that includes all the books or episodes on your server.
Can you elaborate on what you are seeing and what you expect to see?

**Comment by asdil12** [comment]
I am only seeing (names translated from German locale):
Recently added
Current series
Discover
Newest authors
Local books

This is with network access.

So "continue listening" is missing.


But I now tried to completely delete all app data and login again which helped.
I guess it could be related to the app being used with a different server before (which moved, so I did setup a new server in the app). Maybe that created some issues.
